sabnzbd/scheduler.py

```diff
@@ -391,11 +397,24 @@ def plan_diskspace_resume(self, full_dir: str, required_space: float):
             self.__check_diskspace, "check_diskspace", 5 * 60, 9 * 60, "threaded", args=[full_dir, required_space]
         )
 
+    def plan_required_server_resume(self, interval: int = 5):
```

Comment:
I wonder if we should use `plan_resume()` in this case? This way the user gets a visual indication how long the pause will last until it tries again.
This is different from the diskspace pause, because for the diskspace pause, it gives a warning before pausing. But here, the queue would just switch to paused mode and magically unpause again after a while.